test/Dialect/FIRRTL/canonicalization.mlir

```diff
@@ -664,9 +664,9 @@ firrtl.module @Shr(in %in1u: !firrtl.uint<1>,
   firrtl.connect %out1u, %9 : !firrtl.uint<1>, !firrtl.uint<0>
 
   // Issue #6608: https://github.com/llvm/circt/issues/6608
-  // CHECK: firrtl.strictconnect %out1u, %c0_ui1
-  %10 = firrtl.shr %in0u, 0 : (!firrtl.uint<0>) -> !firrtl.uint<1>
-  firrtl.strictconnect %out1u, %10 : !firrtl.uint<1>
+  // CHECK: firrtl.strictconnect %out1s, %c0_si1
+  %10 = firrtl.shr %in0s, 0 : (!firrtl.sint<0>) -> !firrtl.sint<1>
+  firrtl.strictconnect %out1s, %10 : !firrtl.sint<1>
```

Comment:
This test should be updated, not changed. The change makes it identical to the subsequent test.

Reply:
Good catch, will fix